packages/react-shopify-app-route-propagator/src/index.ts

```diff
@@ -1,4 +1,4 @@
-import * as React from 'react';
+import {useState, useEffect} from 'react';
```

Comment:
I've not been destructuring the imports and just doing `React.useState`. Do we have a preference?

Reply:
you can also do `React, {useState, useEffect}`